### ISSUE TITLE
FileNotFoundException when using locations with manual migration

### ISSUE THREAD

**Issue by lunaryorn** [flyway/flyway-play#29]
Hello,

we are using flyway-play with the following configuration in "development mode" (e.g. `sbt run`):

```
db.default.migrations.locations=["prod", "stage"]
db.default.migrations.auto=false
```

Unfortunately flyway-play crashes with a `FileNotFoundException` in this scenario ([stacktrace](https://gist.github.com/lunaryorn/4e862b7a3a5e08c3252401776403ede1)) when pending migrations exist.  Note that the crash does not occur when

- we run our unit tests against the database (e.g. `Mode.Test`),
- when we enable `db.default.migrations.auto=true`,
- when we do not use `locations` at all,
- or when all migrations are successfully applied.

I think the root cause is that `PlayInitializer.migrationDescriptionToShow` does not take `locations` into account when trying to read the migration file for the message.  Hence the error only occurs when the migrations are not applied automatically because `checkState` and thus `migrationDescriptionToShow` are never invoked in this case, or when there are no pending migrations because in this case `checkState` doesn't call `migrationDescriptionToShow`.

**Comment by jboissard** [comment]
hey I ran into same problem but noticed that when I have `db.default.migration.auto = true` the migration works

**Comment by tototoshi** [comment]
Fixed by #33
